### PR TITLE
Adjust colors for confirm/reject/primary buttons

### PR DESCRIPTION
**Pull request by elgatovital**
Handles #1391

@gnosis/mobile-devs

### CODE REVIEW

File: app/src/main/java/io/gnosis/safe/helpers/ConnectivityInfoProvider.kt [hunk 1]
```diff
@@ -12,12 +12,12 @@ class ConnectivityInfoProvider(private val connectivityManager: ConnectivityMana
     var offline: Boolean = true
 
     private val networkCallback = object : ConnectivityManager.NetworkCallback() {
-        override fun onAvailable(network: Network?) {
+        override fun onAvailable(network: Network) {
```
Comment (biafra23) [inline app/src/main/java/io/gnosis/safe/helpers/ConnectivityInfoProvider.kt:15]: This fixes another bug, right? Or was this not meant to be in this PR?

File: build.gradle [hunk 2]
```diff
@@ -35,11 +35,11 @@ subprojects {
     afterEvaluate { project ->
         if (project.hasProperty("android")) {
             android {
-                compileSdkVersion 29
+                compileSdkVersion 30
```
Comment (biafra23) [inline build.gradle:38]: Sneaky!

**Review by biafra23 — APPROVED** [review]
This PR seems to fix more than only colors. If that was intentional. Then: 👍🏻

File: app/src/main/java/io/gnosis/safe/helpers/ConnectivityInfoProvider.kt (hunk 1 above)
Comment (elgatovital) [reply app/src/main/java/io/gnosis/safe/helpers/ConnectivityInfoProvider.kt:15]: no, it was fixed in previous pr with try catch around read-only network request

**Pull request merged**